db/migrations/20221031120819_options.sql

```diff
@@ -0,0 +1,8 @@
+-- migrate:up
+CREATE TABLE options (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR NOT NULL
+)
```

Comment:
rooms에 대한 options가 여러가지인 경우면 room : option = 1 : N 관계로 보여집니다.

```suggestion
CREATE TABLE options {
    id int NOT NULL AUTO_INCREMENT PRIAMARY_KEY,
    name VARCHAR(16) NOT NULL,
    room_id INT
    CONSTRAINT options_room_id_fkey FOREIGN KEY (room_id) REFERENCES (rooms_id)
}

```

Reply:
N : N 관계인 거 같아서 중간테이블을 새로 만들었습니다.